### PR TITLE
bug fix: add int type for rng seed

### LINKED ISSUE #191: TypeError in match_template
Hi, I am trying out the new version release with the phase randomization. I am running this command:

`pytom_match_template.py -t tmpl.mrc --mask mask.mrc -v tomos/bin4_novactf/115.rec -d /scicore/home/engel0006/GROUP/pool-engel/Philippe/rbcPipeline/pytom/current/PyTOM/115/ -a temp/115.tlt --particle-diameter 70.0 -g 0 1 2 3 --voxel-size-angstrom 7.84 --non-spherical-mask --tomogram-ctf-model phase-flip --random-phase-correction --rng-seed 42 --per-tilt-weighting --amplitude-contrast 0.07 --dose-accumulation temp/115_exposure.txt --defocus temp/115_defocus.txt --voltage 300.0 --spherical-aberration 2.7`

and it seems something is wrong with the `--random-phase-correction --rng-seed 42` because it runs fine when I leave out the two flags. Maybe I am parsing this wrong since the type error is between a str and a float (42.0 doesn't fix it). Thanks for the help!

the error: 
Traceback (most recent call last):
  File "/scicore/projects/scicore-p-structsoft/ubuntu/software/Miniconda3/miniconda3_python3.12.1/envs/pytom-tm-ub/bin/pytom_match_template.py", line 8, in <module>
    sys.exit(match_template())
             ^^^^^^^^^^^^^^^^
  File "/scicore/projects/scicore-p-structsoft/ubuntu/software/Miniconda3/miniconda3_python3.12.1/envs/pytom-tm-ub/lib/python3.12/site-packages/pytom_tm/entry_points.py", line 813, in match_template
    args = parser.parse_args(argv)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/scicore/projects/scicore-p-structsoft/ubuntu/software/Miniconda3/miniconda3_python3.12.1/envs/pytom-tm-ub/lib/python3.12/argparse.py", line 1891, in parse_args
    args, argv = self.parse_known_args(args, namespace)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scicore/projects/scicore-p-structsoft/ubuntu/software/Miniconda3/miniconda3_python3.12.1/envs/pytom-tm-ub/lib/python3.12/argparse.py", line 1924, in parse_known_args
    namespace, args = self._parse_known_args(args, namespace)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scicore/projects/scicore-p-structsoft/ubuntu/software/Miniconda3/miniconda3_python3.12.1/envs/pytom-tm-ub/lib/python3.12/argparse.py", line 2136, in _parse_known_args
    start_index = consume_optional(start_index)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scicore/projects/scicore-p-structsoft/ubuntu/software/Miniconda3/miniconda3_python3.12.1/envs/pytom-tm-ub/lib/python3.12/argparse.py", line 2076, in consume_optional
    take_action(action, args, option_string)
  File "/scicore/projects/scicore-p-structsoft/ubuntu/software/Miniconda3/miniconda3_python3.12.1/envs/pytom-tm-ub/lib/python3.12/argparse.py", line 2000, in take_action
    action(self, namespace, argument_values, option_string)
  File "/scicore/projects/scicore-p-structsoft/ubuntu/software/Miniconda3/miniconda3_python3.12.1/envs/pytom-tm-ub/lib/python3.12/site-packages/pytom_tm/io.py", line 75, in __call__
    if values <= 0.0:
       ^^^^^^^^^^^^^
TypeError: '<=' not supported between instances of 'str' and 'float'

### PR DESCRIPTION
**Pull request by McHaillet**
Closes #191

- [x] But should still add a test for it.

### CODE REVIEW

File: tests/test_entry_points.py [hunk 1]
```diff
@@ -157,6 +157,10 @@ def start(arg_dict):  # simplify run
         # phase randomization test
         arguments = defaults.copy()
         arguments["-r"] = ""
+        arguments["-r"] = ""
```
Comment (sroet) [inline tests/test_entry_points.py:160]: seems duplicate from line 159?
```suggestion
```

**Review by sroet — CHANGES_REQUESTED** [review]
One double line of code, LGTM otherwise!

File: tests/test_entry_points.py (hunk 1 above)
Comment (McHaillet) [reply tests/test_entry_points.py:160]: Yup, thanks for spotting

**Review by sroet — APPROVED** [review]
LGTM!

**Pull request merged**